emit-links.js

```diff
@@ -24,7 +22,7 @@ function emitLinks (msg, emit) {
     }
   })
   links.forEach(link => {
-    emit(extend(msg, {
+    emit(Object.assign(msg, {
```

Comment:
I don't think this breaks anything, but *if possible* I think it's usually best to use `Object.assign({}, msg, { .. })` so that we don't mutate `msg`. What do you think?

Reply:
You're right. I thought `extend(msg...)` mutated `msg` but I just checked the `xtend` source and see that by default it doesn't mutate.

Good catch. Updated the PR accordingly.